app/models/user.rb

```diff
@@ -5,4 +5,9 @@ class User < ApplicationRecord
   has_many :books, through: :checkouts
 
   validates :name, presence: true
+
+  def total_fines
+    total = checkouts.inject(0) { |total, checkout| total + checkout.fine }
+    total = sprintf('%.2f', total)
```

Comment:
The formatting here seems like a view concern. Since you're using `number_to_currency` in the view, do you need to format it here?